ui/admin/app/components/form/auth-method/ldap/index.hbs

```diff
@@ -0,0 +1,662 @@
+    @isInvalid={{@model.errors.urls}}
+    @type='text'
+    name='urls'
+    disabled={{form.disabled}}
+    {{on 'input' this.setUrls}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.auth-method.form.urls.label'}}</F.Label>
+    <F.HelperText>{{t 'resources.auth-method.form.urls.help'}}</F.HelperText>
+    {{#if @model.errors.urls}}
+      <F.Error as |E|>
+        {{#each @model.errors.urls as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  <Hds::Form::Fieldset @isOptional={{true}} as |F|>
+    <F.Legend>{{t 'resources.auth-method.form.certificates.label'}}</F.Legend>
+    <F.HelperText>{{t
+        'resources.auth-method.form.certificates.help'
+      }}</F.HelperText>
+    {{#let (unique-id) as |valueHeaderID|}}
+      <F.Control>
+        <Hds::Table class='key-value-list' name='certificates'>
+          <:head as |H|>
+            <H.Tr>
+              <H.Th id={{valueHeaderID}}>{{t 'titles.value'}}</H.Th>
+              <H.Th>{{t 'titles.actions'}}</H.Th>
+            </H.Tr>
+          </:head>
+          <:body as |B|>
+            {{#each @model.certificates as |cert i|}}
+              <B.Tr>
+                <B.Td>
+                  <Hds::Form::Textarea::Field
+                    @value={{cert.value}}
+                    @isOptional={{true}}
+                    disabled={{form.disabled}}
+                    aria-labelledby={{valueHeaderID}}
+                  />
+                </B.Td>
+                <B.Td>
+                  <Hds::Button
+                    @text={{t 'actions.remove'}}
+                    @color='critical'
+                    @icon='trash'
+                    @isIconOnly={{true}}
+                    disabled={{form.disabled}}
+                    {{on 'click' (fn @removeItemByIndex 'certificates' i)}}
+                  />
+                </B.Td>
+              </B.Tr>
+            {{/each}}
+            <B.Tr>
+              <B.Td>
+                <Hds::Form::Textarea::Field
+                  @value={{this.newCertificate}}
+                  @isOptional={{true}}
+                  disabled={{form.disabled}}
+                  aria-labelledby={{valueHeaderID}}
+                />
+              </B.Td>
+              <B.Td>
+                <Hds::Button
+                  @text={{t 'actions.add'}}
+                  @color='secondary'
+                  disabled={{or form.disabled (not this.newCertificate)}}
+                  {{on 'click' this.addCertificate}}
+                />
+              </B.Td>
+            </B.Tr>
+          </:body>
+        </Hds::Table>
+      </F.Control>
+    {{/let}}
+    {{#if @model.errors.certificates}}
+      <F.Error as |E|>
+        {{#each @model.errors.certificates as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::Fieldset>
+
+  {{#if @model.isNew}}
```

Comment:
They can be updated but they return an hmac so when the model is not new then it will use the secret-input component. This will be done in[ ICU-10077](https://hashicorp.atlassian.net/browse/ICU-10077)